lib/PubNub/PubSub/Message.pm

```diff
@@ -81,7 +82,7 @@ Returns the payload encoded in json via Mojo::JSON
 
 sub json {
     my $self = shift;
-    return encode_json($self->{payload});
+    return Encode::encode_utf8($json->encode($self->{payload}));
```

Comment:
Surely now the documentation on line 79 that it is "via Mojo::JSON" is no longer accurate?